src/architecture/msgPayloadDefC/AccessMsgPayload.h

```diff
@@ -33,6 +33,8 @@ typedef struct {
     double az_dot; //!< [rad/s] Azimuth angle rate for a given spacecraft in the SEZ rotating frame.
     double r_BL_L[3]; //!<[m] Spacecraft position relative to the groundLocation in the SEZ frame.
     double v_BL_L[3]; //!<[m/s] SEZ relative time derivative of r_BL vector in SEZ vector components.
+    
+    int pL[2]; //!< [-] pixel of the location as view from pinhole camera
```

Comment:
Mm, I don't see why this pixel info would be in the access msg?  I prefer that this pixel info is put into its own message type.

Reply:
Yeah, it was 50-50 decision to me, glad that you've made it. I've created a separate LandmarkMsgPayload.h message and reverted the access one.